index.js

```diff
@@ -62,7 +77,16 @@ function game() {
   let finalScore = 100;
 
   while (counter < maxAttempts) {
-    let playerGuess = getPlayerGuess();
+    let playerGuess = getPlayerGuess(finalScore);
+
+    while (playerGuess === null) {
```

Comment:
Could you do this without a loop? It's not a good idea to have a loop inside a loop. It's bad practice so I would much prefer it if we could find another solution. But this assignment is not about performance I guess so if you can't find another solution, I would be okay with this